instance/models/openedx_instance.py

```diff
@@ -146,6 +147,21 @@ def studio_domain(self):
             return self.internal_studio_domain
 
     @property
+    def studio_domain_nginx_regex(self):
+        """
+        Regex that matches either the internal or the external Studio URL.
+
+        This is exclusively meant for the Ansible variable CMS_HOSTNAME to configure the nginx
+        server_name regex to match the Studio domains.
+        """
+        if self.external_studio_domain:
+            domains = [self.external_studio_domain, self.internal_studio_domain]
+        else:
+            domains = [self.internal_studio_domain]
+        choices = '|'.join(map(re.escape, domains))  # pylint: disable=bad-builtin
```

Comment:
@smarnach @itsjeyd Pylint was complaining about `map` here, right? Was there a reason for not replacing it with a list comprehension rather than silencing it?